react-client/src/components/Common/AddButton/AddButton.module.css

```diff
@@ -0,0 +1,42 @@
+@import "https://use.fontawesome.com/releases/v5.5.0/css/all.css";
+
+.addNewBtn {
+    background-color: #222831;
+    border: none;
+    color: #11998e;
+    filter: drop-shadow(0px 0px 1px);
+    height: 0;
+    padding: 0;
+    position: absolute;
+    right: 60px;
+    top: 15px;
+
```

Comment:
you can just write &:hover{
}
here.
come see me regarding css modules. I think I better explain a few things